code/modules/paperwork/paper.dm

```diff
@@ -158,13 +158,13 @@
 
 /obj/item/paper/examine(mob/user)
 	. = ..()
-	if(in_range(user, src) || isobserver(user))
-		if(user.is_literate())
-			ui_interact(user)
-		else
-			. += "<span class='warning'>You cannot read it!</span>"
-	else
+	if(!in_range(user, src) && !isobserver(user))
 		. += "<span class='warning'>You're too far away to read it!</span>"
+		return
```

Comment:
Just return the string, i'm pretty sure it's faster.

Reply:
I can't with examine because it does a bunch of stuff ahead of the paper...er humm let me check what it does.  There was some reason I had to do this.

Reply:
Sorry, didn't see it was +=.

Reply:
Ah ok its because of the door.  The door chains the examine command if its stuck to it so you have the door description before it and that is added to the end of it.

Reply:
ugh lost my comment.  When its attached to a door the door uses the examine to combine with its message.  That is it calls the paper examine in a chain.  Sorry tired so I will have to review it again tommow.